### ISSUE TITLE
A few questions about fine-tuning

### ISSUE THREAD

**Issue by kellymio** [slei109/PATNet#41]
Thanks for your work! 
Where did the query image come from when you used it for fine-tuning? In addition, for all the tested datasets, do they use the Pascal dataset as the training set and FSS as the validation set?

**Comment by slei109** [comment]
The query image was randomly sampled from the tested datasets. Since we took the average performance of each episode, it won't affect the evaluation results.
Yes, we adopted the Pascal dataset as the training set for all the tested datasets.
